docs/codebase-best-practices.md

```diff
@@ -1,5 +1,23 @@
 # Codebase Best Practices
 
+## Styling a component
+
+Be mindful of adding new colors to the codebase, as you will have to account for contrast and color blindness. Because of this, we always use already determined variables in [variable.css](/client/src/components/layouts/variables.css).
+
+We are against adding new variables to [variable.css](/client/src/components/layouts/variables.css), because we have to account for contrast, color blindness 🙂, and cluttering the devtools which affects negatively the devtools experiences and add concerns for accessibility.
```

Comment:
These first two paragraphs seem like they're kind of saying the same thing. Maybe combine them into one? I would maybe just mention to use the colors from the variables.css file - and you could maybe link to the design style guide.